### ISSUE TITLE
Capistrano v3 compatibility?

### ISSUE THREAD

**Issue by tardate** [kalys/capistrano-nginx-unicorn#14]
Seems this gem is not compatible with capistrano v3, but maybe I missed something.

Are there any plans (or forks) to get this working with v3?

**Comment by bunnymatic** [comment]
I've tried it with Cap3 and it doesn't seem to trigger the restart commands.  I'm going to look deeper to see if i can figure it out, but i'd love to know if it's supported and it's my fault or if it's just not supported.

And i've tried running the tasks manually

```
  % bundle exec cap acceptance deploy unicorn:restart
  cap aborted!
  Don't know how to build task 'unicorn:restart'
```

:(

I did have luck with this gem - https://github.com/tablexi/capistrano3-unicorn - it doesn't have the nginx support but with a tiny bit of extra config, it works out of the box.

thanks, in advance
